src/distributors/zulu-installer.ts

```diff
@@ -0,0 +1,77 @@
+import * as core from '@actions/core';
+import * as tc from '@actions/tool-cache';
+
+import path from 'path';
+import fs from 'fs';
+import semver from 'semver';
+
+import { JavaInstallerResults, JavaDownloadRelease, JavaBase, JavaInstallerOptions } from './base-installer';
+import { IZulu, IZuluDetailed } from './zulu-models';
+import { IS_WINDOWS, IS_MACOS, PLATFORM } from '../util';
+
+// TO-DO: validate feature
+export class ZuluDistributor extends JavaBase {
+    private extension = IS_WINDOWS ? 'zip' : 'tar.gz';
+    private platform: string;
+    constructor(initOptions: JavaInstallerOptions) {
+        super("Zulu", initOptions);
+        this.platform = IS_MACOS ? 'macos' : PLATFORM;
+        if (this.arch === 'x64') {
+            // put small comment
```

Comment:
Please correct this comment with description why we use this hack

Reply:
Done. Thank you.